src/main/java/util/Validator.java

```diff
@@ -0,0 +1,36 @@
+package util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class Validator {
+    public String checkLetter(BufferedReader reader) throws IOException {
+        String input = reader.readLine();
+        while (!(input.equals("r") || input.equals("b") || input.equals("s"))){
+            input = reader.readLine();
+        }
+        return input;

```

Comment:
this method has 2 responsibilities( read input and validate it) and we should stick to single responsibility rule.
I suggest this method will take string input and validate it, so you will be able to keep logic of taking user input in separate place